### DISCUSSION TITLE
Spectre.Console.Cli API reference missing from website

### DISCUSSION THREAD

**Discussion by ivml** [spectreconsole/spectre.console#832] [General]
Since `Spectre.Console.Cli` was moved into its own package in #827 , the API reference pages for both `Spectre.Console.Cli` and `Spectre.Console.Cli.Unsafe` have been missing.

https://spectreconsole.net/api/

![image](https://user-images.githubusercontent.com/4217267/168796291-6bfa81cd-866e-459f-9a48-f21993926a34.png)

Adding the new project to the list of source files in `docs/Program.cs` seems to solve the issue:

![image](https://user-images.githubusercontent.com/4217267/168796448-95502248-b232-4d2b-b32e-0ef8b67fb579.png)

I would have made a PR with the fix directly, but I'm just following the guide to contributing.

**Comment by phil-scott-78** [comment]
Good catch! Get a that PR submitted and I'll give it a review this afternoon.

**Comment by patriksvensson** [comment]
I've created an issue for this from this discussion: https://github.com/spectreconsole/spectre.console/issues/833
